src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.UnifiedBuild.Tasks/RemoveSectionsFromNugetConfig.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
It would be good to #nullable enable

Reply:
yes please - we're hoping to start moving this way for the whole sdk in the more quiet time after the 9.0 release, but it would be good social practice for new code to be nullable from the beginning!